envbuilder_test.go

```diff
@@ -17,36 +17,3 @@ func TestDefaultWorkspaceFolder(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, envbuilder.EmptyWorkspaceDir, dir)
 }
-
-func TestSystemOptions(t *testing.T) {
```

Comment:
Up to you, but I think it makes sense to keep this around and test:
- That all options have a sensible default value
- That all options can be overridden

Reply:
I feel I would be testing Cobra stuff, but I will put more thoughts on that for sure.